### ISSUE TITLE
Search containers by name

### ISSUE THREAD

**Issue by PhlppKrmr** [amosproj/amos2023ss04-kubernetes-inventory-taker#33]
## User story
1. As a user
2. I want / need search containers by their name
3. So that I can easily find them

## Acceptance criteria
- [x] Results are displayed in a list view
- [x] Search is triggered only after pressing enter or the corresponding icon
- [x] The search is always queried against the database
- [x] The search field shall be on the top right, right on top of the list
- [x] The search field is displayed inside the "All Containers" view

## Definition of done (DoD)
* Colours match with [color scheme](https://amos-p4.atlassian.net/l/cp/g5VzjL88)
* Implementation is unit tested
* Code documentation updated (jsdoc or godoc?)
* Changes to database documented at [Confluence- Database scheme](https://amos-p4.atlassian.net/l/cp/zY8cqaaP)
* Changes to architecture documented at [Confluence - Architecture](https://amos-p4.atlassian.net/l/cp/RT8RaER0)

## DoD general criteria
* Feature has been fully implemented
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Pull request to "develop" branch is merged & closed
* Bill of materials is updated with new dependencies

**Comment by jandegen** [comment]
Searching for name and image is possible, great!

**Comment by freefreeP** [comment]
This feature is missing tests, but it got merged into develop because I accidentally had it not as draft PR
